Library/Homebrew/cask/dsl.rb

```diff
@@ -230,8 +230,13 @@ def version(arg = nil)
     end
 
     # @api public
-    def sha256(arg = nil)
-      set_unique_stanza(:sha256, arg.nil?) do
+    def sha256(arg = nil, arm: nil, intel: nil)
+      should_return = arg.blank? && arm.blank? && intel.blank?
```

Comment:
I think these should still check for `nil?`.

Calling `sha256 ""` will simply return instead of setting the checksum to an empty string, so will audit fail correctly?

Reply:
Ahh, good point 👍